src/funmirtar/utils/data_processing.py

```diff
@@ -0,0 +1,94 @@
+from Bio import SeqIO
+import pandas as pd
+
+
+def get_ids_without_version(id_x):
```

Comment:
What is this function doing? It is not used anywhere.

Reply:
Used in the other part of data preprocessing. I will leave it here for now, since soon we will push parts of code that use it